jnosql-mapping/jnosql-mapping-reflection/src/main/java/org/eclipse/jnosql/mapping/reflection/DefaultGroupEntityMetadata.java

```diff
@@ -23,15 +24,19 @@
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
 
-
+/**
+ * The default implementation of {@link GroupEntityMetadata}.
+ * It will load all Classes and put in a {@link ConcurrentHashMap}.
+ * Where the key is {@link Class#getName()} and the value is {@link EntityMetadata}
+ */
 @ApplicationScoped
-public class ReflectionGroupEntityMetadata implements GroupEntityMetadata {
+public class DefaultGroupEntityMetadata implements GroupEntityMetadata {
```

Comment:
I would keep it as `ReflectionGroupEntityMetadata`, because, in the end, that is what it does: read entity metadata by reflection.

Reply:
:+1:  I'll rename it!